packages/react-components/react-toast/src/components/ToastTrigger/ToastTrigger.types.ts

```diff
@@ -0,0 +1,23 @@
+import { ARIAButtonResultProps, ARIAButtonType } from '@fluentui/react-aria';
+import type { TriggerProps } from '@fluentui/react-utilities';
+import * as React from 'react';
+
+export type ToastTriggerProps = TriggerProps<ToastTriggerChildProps> & {
+  /**
+   * Disables internal trigger mechanism that ensures a child provided will be a compliant ARIA button.
+   * @default false
+   */
+  disableButtonEnhancement?: boolean;
```

Comment:
How about we don't introduce this property for this trigger? 🙈 How about we do something more like what would be ideal: don't enhance by default and add a `enableButtonEnhancement`?!

To get back in the history of this property: The default behaviour of a trigger should be to assume that what is given to it is a button, so no enhancement should happen by default, buuuut since this would be a breaking change we introduced `disableButtonEnhancement` and added it everywhere in our docs 😅.